hw4/03-form.js

```diff
@@ -1,18 +1,46 @@
 const express = require('express');
+
 const app = express();
 const port = process.env.PORT || 5000;
 
 // Use middleware static() to serve all static files in the given folder
 app.use(express.static('public'));
 
-// Use middleware urlencoded() to parse an incoming request with a urlencoded payload and return an objectß
+// Use middleware urlencoded() to parse an incoming request with a urlencoded
+// payload and return an objectß
 app.use(express.urlencoded({ extended: false }));
 
-// POST request
-app.post('', (req, res) => {
-  // Add your code here
-});
+app.use('/', express.static('public'));
 
-app.listen(port, () => {
-  console.log(`Server running at http://localhost:${port}`);
+const getFormResultsHTML = (body) => {
+  let result = '';
+  let wantsNewsletter = false;
+  const keys = Object.keys(body);
+  for (let i = 0; i < keys.length; i += 1) {
+    const key = keys[i];
+    let val = body[key];
```

Comment:
The empty string is falsy.
I'm seeing this pattern of sometimes setting a value and sometimes using a default value
The logical or operator can help you perform simple conditional logic in your variable initialization.

here you could say: 
```javascript
    let val = body[key] || 'n/a';
```